pyaedt/edb_core/siwave.py

```diff
@@ -220,12 +247,18 @@ def rvalue(self, value):
 
     @property
     def source_type(self):
+        """Source type."""
         return self.source_type
 
 
 class EdbSiwave(object):
-    """EdbSiwave class.
+    """EDB Siwave application interface.
 
+    Parameters
+    ----------
+    parent : str
```

Comment:
```suggestion
    parent :
```
Since it is an object.